packages/services/src/contents/index.ts

```diff
@@ -1080,8 +1115,21 @@ export class Drive implements Contents.IDrive {
     this._apiEndpoint = options.apiEndpoint ?? SERVICE_DRIVE_URL;
     this.serverSettings =
       options.serverSettings ?? ServerConnection.makeSettings();
+    const restContentProvider = new RestContentProvider({
+      apiEndpoint: this._apiEndpoint,
+      serverSettings: this.serverSettings,
+      fileChanged: this.fileChanged
```

Comment:
RTC needs to emit `fileChanged` when the file is saved from server-sid. It can currently do this because it is swapping the drive. Once it is just provisioning the content via content provider it would no longer be able to emit `fileChanged`.

I agree that the implementation should be improved - instead of passing down the signal to emit from provider, we should allow content providers to emit `fileChanged` and proxy that in the drive.

Reply:
Done in df6a62ff02ae4ed314b98200cec34a532e13d124